Wox.Plugin.Runner/Runner.cs

```diff
@@ -12,8 +12,8 @@ namespace Wox.Plugin.Runner
 {
     public class Runner : IPlugin, ISettingProvider
     {
-        internal static PluginInitContext Context;
-        RunnerSettingsViewModel viewModel;
+        internal static PluginInitContext? Context;
```

Comment:
About the Nullity check for this one. I suggest making this non-nullable, since it is actually initialized in a "Constructor" like function. Maybe just put something like `internal static PluginInitContext Context = null!;` to indicate the situation.